Caleydo/coral
# Client Config

Closes https://github.com/Caleydo/cohort/issues/669

The contact if an error occurs is now configurable during runtime with a [clientConfig.json](https://docs.google.com/presentation/d/1k36d_CGs5euOdk73AUNC54mWKzztud17ltzkxhM-KU0/edit#slide=id.gb2dc75320b_0_8):
![image](https://user-images.githubusercontent.com/10337788/159361722-8345844d-ffeb-4ecc-8c32-8f9ecb4a2abe.png)


To test:
* make sure you can't retrieve the data (e.g., home office without vpn, or switch to eduroam @ JKU)
* in your default environment. the current behavior should stay as it is and display:
	![image](https://user-images.githubusercontent.com/10337788/159361938-c50a3c32-5566-4158-b870-001ac5fe3aaa.png)
* different outputs via clientConfig settings:
	*  checkout the _clientConfig_ branch of _coral_product_: `git@github.com:Caleydo/coral_product.git -b clientConfig`
	* install dependencies `npm install`
	* build docker images: `npm run build`
	* start the containers with `docker-compose up`
	* visit http://localhost/app which should display: `contact John Smith` instead of `report an issue` after the data could not be retrieved
	* update the clientConfig.json inside the coral_product repo, save,  and reload the page --> the changes should be reflected
